components/ProductGrid/components/Item/index.jsx

```diff
@@ -37,59 +37,91 @@ const Item = ({ product, display, isFavorite }) => (
+    <Portal name={portals.PRODUCT_ITEM_FAVORITES_BUTTON_BEFORE} props={{ productId: product.id }} />
+    <Portal name={portals.PRODUCT_ITEM_FAVORITES_BUTTON} props={{ productId: product.id }}>
+      <div className={styles.favorites}>
+        <FavoritesButton active={isFavorite} productId={product.id} noShadow removeWithRelatives />
       </div>
-    }
-    <div className={styles.favorites}>
-      <FavoritesButton active={isFavorite} productId={product.id} noShadow removeWithRelatives />
-    </div>
+    </Portal>
+    <Portal name={portals.PRODUCT_ITEM_FAVORITES_BUTTON_AFTER} props={{ productId: product.id }} />
+
     {(!display || display.name || display.price || display.reviews) && (
       <div className={styles.details}>
-        <Portal name={PRODUCT_ITEM_NAME_BEFORE} props={{ id: product.id }} />
+
+        {/* NAME */}
         {(!display || display.name) && (
-          <div className={styles.title} itemProp="name">
-            <Ellipsis>{product.name}</Ellipsis>
-          </div>
-        )}
-        {(!display || display.price) && (
-          <Grid className={styles.priceWrapper} wrap>
-            <Grid.Item grow={1}>
-              <Price
-                unitPrice={product.price.unitPrice}
-                unitPriceMin={product.price.unitPriceMin}
-                discounted={!!product.price.discount}
-                currency={product.price.currency}
-              />
-            </Grid.Item>
-            {product.price.msrp > 0 && (
-              <Grid.Item>
-                <PriceStriked
-                  value={product.price.msrp}
-                  currency={product.price.currency}
-                />
-              </Grid.Item>
-            )}
-            {(!product.price.msrp && product.price.unitPriceStriked > 0) && (
-              <Grid.Item>
-                <PriceStriked
-                  value={product.price.unitPriceStriked}
-                  currency={product.price.currency}
-                />
-              </Grid.Item>
-            )}
-          </Grid>
+          <Fragment>
+            <Portal name={portals.PRODUCT_ITEM_NAME_BEFORE} props={{ productId: product.id }} />
+            <Portal name={portals.PRODUCT_ITEM_NAME} props={{ productId: product.id }}>
+              <div className={styles.title} itemProp="name">
+                <Ellipsis>{product.name}</Ellipsis>
+              </div>
+            </Portal>
+            <Portal name={portals.PRODUCT_ITEM_NAME_BEFORE} props={{ productId: product.id }} />
+          </Fragment>
         )}
+
+        {/* PRICE - STRIKE PRICE - PRICE INFO */}
         {(!display || display.price) && (
-          <Grid>
-            {product.price.info && (
-              <Grid.Item>
-                <PriceInfo className={styles.basicPrice} text={product.price.info} />
-              </Grid.Item>
-            )}
-          </Grid>
+          <Fragment>
+            <Portal name={portals.PRODUCT_ITEM_PRICE_BEFORE} props={{ productId: product.id }} />
```

Comment:
There are no PRODUCT_ITEM_PRICE_XXX constants in common-commerce

Reply:
Now there are